README.md

```diff
@@ -20,9 +20,31 @@ WIKI: https://wiki.mozilla.org/Idea-Town
 
 * [Install Docker](http://docs.docker.com/mac/started/) and docker-compose
 
-* `docker up`
+* Make sure you have a default Docker machine, i.e.:
+  * `docker-machine create -driver virtualbox default`
 
-* Visit http://0.0.0.0:8000
+* Make sure your shell can see the default Docker machine, i.e.:
+  * `eval "$(docker-machine env default)"`
+
+* Create & set up the Docker containers:
+  * `docker-compose up`
+  * This may take some time.
+
+* To see the IP address of the default Docker machine:
+  * `docker ip default` 
+
+* To visit the Django server:
+  * `open http://$(docker-machine ip default):8000/`
+  * (or, whatever IP was reported by `docker ip` and port 8000)
+
+* To shell into one of the containers, e.g. to run Django commands:
+  * `docker exec -t -i ideatown_server_1 bash`
```

Comment:
Whoa, that's odd. I'll try creating a fresh repo and see which names are listed.